networks

```diff
@@ -17,7 +17,7 @@ harmony
 evmos
 aurora
 boba
+nova
+
 goerli
-sepolia 
```

Comment:
I removed sepolia from the list for now, as we don't have our contract deployed there.